Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -401,6 +461,34 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this.color = resolvedColor;
         this.dispatchEventToListeners(WI.ColorPicker.Event.ColorChanged, {color: this._color, variableName});
     }
+
+    _updateColorContrast(color)
+    {
+        if (this._activeSwatch && color) {
+            this._activeSwatch.value = color;
+            this._activeSwatch.element.style.backgroundColor = color;
+            if (this._contrastObject.active === "background-color")
+                this._contrastObject.propertyBackgroundColor = color;
+            else
+                this._contrastObject.propertyForegroundColor = color;
+        }
+
+        if (this._contrastScore) {
+            let contrast = WI.ColorPicker.accessabilityContrast(this._contrastObject.propertyBackgroundColor, this._contrastObject.propertyForegroundColor);
+            contrast = Math.ceil(contrast * 100) / 100;
+            let contrastStatus = `Color Contrast: ${contrast} | `;
+            if (contrast >= 7) {
+                contrastStatus += "AAA";
+            } else if (contrast >= 4.5) {
+                contrastStatus += "AA (large text)";
+            } else if (contrast >= 3) {
+                contrastStatus += "AA (regular text)";
+            } else {
+                contrastStatus += "Fail";
```

Comment:
Be sure to add a descriptive comment for translators so they understand the context of where it's used and what the opposite of `Fail` is in this context. The word `Pass` doesn't appear in the alternatives so they can derive meaning from it.

For example: "Label shown when the color contrast between two colors does not pass the minimum required threshold".